### DISCUSSION TITLE
By mistake I have deleted the pull requests status tracking Issue.

### DISCUSSION THREAD

**Discussion by MrBotDeveloper** [renovatebot/renovate#11604] [Request Help] [answered]
As per I have described in the title By mistake I have deleted the pull requests status tracking Issue. How can I get it back a **new same issue opened by the bot** ?

My Repo=> https://github.com/MrBotDeveloper/PDF-Bot

**Comment by viceice** [comment, marked as answer]
It should be recreated by next bot run. Check https://app.renovatebot.com

**Reply by viceice** [reply]
I assume you speak about the dependency dashboard

**Reply by MrBotDeveloper** [reply]
Ya thanks now its automatically created 👍
